src/main/java/com/linkedin/xinfra/monitor/apps/SingleClusterMonitor.java

```diff
@@ -94,14 +97,9 @@ public void start() throws Exception {
       _topicManagementService.start();
       CompletableFuture<Void> topicPartitionResult = _topicManagementService.topicPartitionResult();
 
-      try {
       /* Delay 2 second to reduce the chance that produce and consumer thread has race condition
       with TopicManagementService and MultiClusterTopicManagementService */
-        long threadSleepMs = TimeUnit.SECONDS.toMillis(2);
```

Comment:
I tried to concisely use the `Utils.delay(...)`  method. Here, I additionally had to import `delay` statically since a Kafka `Utils` class is already used. (avoiding fully qualified import)